force-app/main/default/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -191,6 +191,12 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
             dynamicSoql += 'CurrencyIsoCode, ';
         }
 
+        Set<String> objectFields = Schema.SObjectType.Contact.fields.getMap().keySet();
+        Boolean ispersonNameSuffixEnabled = objectFields.contains('suffix');
+        if (ispersonNameSuffixEnabled) {
+            dynamicSoql += 'Suffix, ';
+        }
+
```

Comment:
Could you please clarify what is the purpose of this change to TDTM? I thought this is just a test automation story

Reply:
These changes are required to automate [T-3904064](https://gus.lightning.force.com/lightning/r/ADM_Test_Scenario__c/a80B00000001ZeJIAU/view)/[T-3904063](https://gus.lightning.force.com/lightning/r/ADM_Test_Scenario__c/a80B00000001ZeEIAU/view) as person name suffixes will currently result in an error.

Current behavior:

https://user-images.githubusercontent.com/16750742/152391837-219c74d6-9a46-4a46-8543-cbfe7f8acc0a.mp4


Proposed behavior:


https://user-images.githubusercontent.com/16750742/152391888-5dc12ddc-5984-4f83-8b5c-35c665b3e6b3.mp4

Reply:
Interesting, good point Iskander and Thanks @ZeeshanSyedH  for documenting it. 
I wonder if there is a known bug reported already about Suffixes not working correctly. I believe that in terms of the scope of this automation we possibly should not fix any bugs as well? I wonder if this is something we should bring up in our next stand up and let Jason and other devs decide.

Reply:
I agree with Sal. I tested this change and it fixes the suffix issue indicated on the video here. I think we should discuss this with Jason

Reply:
Agreed, lets take it up with product. For what its worth, the possibility of using suffix is mentioned within the product documentation currently:
![Screen Shot 2022-02-03 at 7 23 29 PM](https://user-images.githubusercontent.com/16750742/152451625-75d4b4f1-4906-4fb9-9af2-546fd2691360.png)